02_housing/2a_affordable_available_place.R

```diff
@@ -161,7 +161,7 @@ available_2022_subgroup <- available_2022_final %>%
   # remove leading underscore and capitalize words
   mutate(subgroup = str_remove(subgroup, "_") %>% str_to_title(),
          subgroup_type = "tenure") %>% 
-  # join with affordable values for final file
```

Comment:
My understanding (from Claudia) is that these final files should match the format of the previous final files. I believe this means that what is currently rate_ should be switched to named share_ and these affordable values do not need to be merged on since they already exist in the other files.